### PR TITLE
podcast hub and streams

### PR DESCRIPTION
**Pull request by i-like-robots**
Adds latest episodes to podcast hub and podcast streams. This copies the story card component from the front page.

## Hub
screen shot 2015-10-01 at 14 43 59

## Stream
screen shot 2015-10-01 at 14 43 15

<small>* yes I know the timestamps on the cards in the screenshots are borked, they ain't now</small>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "ft-poller": "^2.0.3",
     "html-to-text": "^1.3.0",
     "intl": "^1.0.0",
+    "n-podcast-mapping": "Financial-Times/n-podcast-mapping",
```
Comment (ironsidevsquincy) [inline package.json]: version?

File: server/models/article-v1.js [hunk 2]
```diff
@@ -10,6 +9,7 @@ module.exports = function(article, opts) {
 	opts = opts || {};
 	var output = { raw: article };
 	output.id = article.item.id;
+	output.url = '/content/' + article.item.id;
```
Comment (ironsidevsquincy) [inline server/models/article-v1.js]: template string!

File: server/controllers/v1-taxonomy.js [hunk 3]
```diff
@@ -129,37 +132,60 @@ module.exports = function(req, res, next) {
...
-									hotOpinion = items.find(function(item) {
-										return !item.template && itemGenreIs(item, 'Comment');
-									});
-									if (hotOpinion && !hotOpinion.image) {
-										hotOpinion.image = extractImage(hotOpinion);
-									}
+							// Podcast hubs
+							if (res.locals.flags.podcasts) {
+								let podcastShowTags = podcastMapping.primaryTags();
+								let podcastHubId = 'NjI2MWZlMTEtMTE2NS00ZmI0LWFkMzMtNDhiYjA3YjcxYzIy-U2VjdGlvbnM=';
```
Comment (ironsidevsquincy) [inline server/controllers/v1-taxonomy.js]: `const`s

File: package.json (hunk 1 above)
Comment (i-like-robots) [inline package.json]: There isn't one =)

**Pull request merged**